runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClient.java

```diff
@@ -198,27 +220,33 @@ public BundleProcessor call() {
    * <p>A client may block on the result future, but may also proceed without blocking.
    */
   public Map<String, BundleProcessor> register(
-      Iterable<BeamFnApi.ProcessBundleDescriptor> processBundleDescriptors) {
+      Map<BeamFnApi.ProcessBundleDescriptor, RemoteInputDestination<WindowedValue<?>>>
+          processBundleDescriptors) {
 
+    LOG.info("Registering {}", processBundleDescriptors.keySet());
```

Comment:
This is going to be quite verbose, debug?

Reply:
Done.